### PR TITLE
ErrorTripSwitch v3

### PR DESCRIPTION
**Pull request by kcrobinson**
@smarts @pikadanx

### CODE REVIEW

**Comment by smarts** [comment]
:+1:

File: src/log4net.CircuitBreakerErrorHandler.UnitTests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("log4net.CircuitBreakerErrorHandler.UnitTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (pikadanx) [inline src/log4net.CircuitBreakerErrorHandler.UnitTests/Properties/AssemblyInfo.cs]: `InfoSpace, LLC`

File: src/log4net.CircuitBreakerErrorHandler.UnitTests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyProduct("log4net.CircuitBreakerErrorHandler.UnitTests")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```
Comment (pikadanx) [inline src/log4net.CircuitBreakerErrorHandler.UnitTests/Properties/AssemblyInfo.cs]: `Copyright © InfoSpace, LLC`

File: src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("log4net.ErrorHandlerCircuitBreaker")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("log4net.ErrorHandlerCircuitBreaker")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```
Comment (pikadanx) [inline src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs]: `Copyright © InfoSpace, LLC`

File: src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("log4net.ErrorHandlerCircuitBreaker")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (pikadanx) [inline src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs]: `InfoSpace, LLC`

File: src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (pikadanx) [inline src/log4net.ErrorHandlerCircuitBreaker/Properties/AssemblyInfo.cs]: version 0.1.0.0

**Comment by pikadanx** [comment]
other than assembly infos and version; :+1:

**Pull request merged**